client/my-sites/customer-home/go-mobile-card/index.jsx

```diff
@@ -32,6 +33,7 @@ export const GoMobileCard = ( { translate, email, sendMobileLoginEmail } ) => {
 	const showIosBadge = isDesktopView || isIos || ! isAndroid;
 	const showAndroidBadge = isDesktopView || isAndroid || ! isIos;
 	const showOnlyOneBadge = showIosBadge !== showAndroidBadge;
+	const isDesktopApp = [ 'desktop', 'desktop-development' ].includes( config( 'env_id' ) );
```

Comment:
We have a `desktop` feature flag that is only enabled on the desktop app, so we could use that instead.
```suggestion
	const isDesktopApp = config.isEnabled( 'desktop' );
```

Reply:
oh, nice 👍

Reply:
oh, nice 👍